src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Isang pambihirang pagpaptunay ang nangyari.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Hindi mahanap ang mga kinakailangan na dokumento para sa pagpapatunay.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <target>Hindi maproseso ang iyong hiling dahil may problema sa sistema.</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>Invalid credentials.</source>
+                <target>Hindi balidong mga dokumento.</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>Cookie has already been used by someone else.</source>
+                <target>Ang Cookie ay ginamit na ng ibang tao.</target>
+            </trans-unit>
+            <trans-unit id="6">
+                <source>Not privileged to request the resource.</source>
+                <target>Walang pribilehiyo upang humingi ng mga bagong mapagkukunan.</target>
+            </trans-unit>
+            <trans-unit id="7">
+                <source>Invalid CSRF token.</source>
+                <target>Hindi balidong mga token ng CSRF.</target>
+            </trans-unit>
```

Comment:
number 8 is missing https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Security/Core/Resources/translations/security.en.xlf#L33 , required for 2.7

Reply:
Sir, I already added the missing trans-unit #8 ...